open-metadata-resources/open-metadata-archives/open-metadata-types/src/main/java/org/odpi/openmetadata/opentypes/OpenMetadataTypesArchive.java

```diff
@@ -217,6 +218,1635 @@ private ClassificationDef addTemplateClassification()
+
+        property = archiveHelper.getStringTypeDefAttribute(attribute1Name,
+                attribute1Description,
+                attribute1DescriptionGUID);
+        property.setAttributeStatus(TypeDefAttributeStatus.DEPRECATED_ATTRIBUTE);
+        property.setReplacedByAttribute(attribute1ReplacedBy);
+        properties.add(property);
+
+        final String attribute2Name            = "deployedImplementationType";
+        final String attribute2Description     = "Type of implemented or deployed software server.";
+        final String attribute2DescriptionGUID = null;
+
+        property = archiveHelper.getStringTypeDefAttribute(attribute2Name,
+                attribute2Description,
+                attribute2DescriptionGUID);
+
+        properties.add(property);
+
+        final String attribute3Name            = "version";
+        final String attribute3Description     = "Deprecated attribute. Use the serverVersion attribute to define the version number of software server.";
+        final String attribute3DescriptionGUID = null;
+        final String attribute3ReplacedBy      = "serverVersion";
+
+        property = archiveHelper.getStringTypeDefAttribute(attribute3Name,
+                attribute3Description,
+                attribute3DescriptionGUID);
+        property.setAttributeStatus(TypeDefAttributeStatus.DEPRECATED_ATTRIBUTE);
+        property.setReplacedByAttribute(attribute3ReplacedBy);
+        properties.add(property);
+
+        final String attribute4Name            = "serverVersion";
+        final String attribute4Description     = "Version number of the software server.";
+        final String attribute4DescriptionGUID = null;
+
+        property = archiveHelper.getStringTypeDefAttribute(attribute4Name,
+                attribute4Description,
+                attribute4DescriptionGUID);
+
+        properties.add(property);
+
+
+        typeDefPatch.setPropertyDefinitions(properties);
+        return typeDefPatch;
+    }
+
+
+    /**
+     * Deprecate clashing properties and add new ones to replace them.
+     * @return the type def patch
+     * @see #updateSoftwareServerPlatformEntity()
+     */
+    private TypeDefPatch updateSoftwareServerCapabilityEntity()
+    {
+        /*
+         * Create the Patch
+         */
+        final String typeName = "SoftwareServerCapability";
+
+        TypeDefPatch  typeDefPatch = archiveBuilder.getPatchForType(typeName);
+
+        typeDefPatch.setUpdatedBy(originatorName);
+        typeDefPatch.setUpdateTime(creationDate);
+
+        /*
+         * Build the attributes
+         */
+        List<TypeDefAttribute> properties = new ArrayList<>();
+        TypeDefAttribute       property;
+
+        final String attribute1Name            = "type";
+        final String attribute1Description     = "Deprecated attribute. Use the capabilityType attribute to describe the type of software server capability.";
```

Comment:
Should this description refer to deployedImplementationType rather than capabilityType, or should the attribute's replacedBy property refer to capabilityType?